tests/fe/interpolate_q_system_mixed_01.debug.output

```diff
@@ -8,7 +8,7 @@ DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)-FE_DGP<1>(1)], Mask 2, P_2, rel. error
 DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)-FE_DGP<1>(1)], Mask 1, P_3, rel. error=0.000296
 DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)-FE_DGP<1>(1)], Mask 2, P_3, rel. error=0.000296
 DEAL::FESystem<1>[FE_Q<1>(1)^2-FE_Q<1>(2)-FE_DGP<1>(1)], Fails for including non-interpolating FE OK
-DEAL::	Fails with ExcNonInterpolatingFE()
+DEAL::	Fails with ExcMessage("The component mask supplied to " "VectorTools::interpolate selects a " "non-interpolatory element.")
```

Comment:
In spirit of our crusade specialized typed exceptions, I simply replaced `ExcNonInterpolatingFE` with an `ExcMessage`.

Reply:
Appreciated! (In reference to #610 :-) )